app/views/users/show.html.haml

```diff
@@ -0,0 +1,37 @@
+%section.user.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %article
+        %header
+          %img(src="http://www.gravatar.com/avatar/#{Digest::MD5.hexdigest(@user.email.downcase)}?s=40")
+          %h2= "#{@user.full_name}'s video collections (#{@user.queued_videos.count})"
+          - if !current_user.has_follower? @user
+            = button_to "Follow", create_following_path(follow_id: @user), class: "btn btn-default"
+        %table.table
+          %thead
+            %tr
+              %th(width="30%") Video Title
+              %th(width="15%") Genre
+          %tbody
+            - @user.queued_videos.each do |queued_video|
+              %tr
+                %td
+                  = link_to queued_video.video_title, queued_video.video, id: queued_video.video.id
+                %td
+                  = link_to queued_video.video_category_title
+
+%section.user_reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %header
+        %h3= "#{@user.full_name}'s Reviews (#{@user.reviews.count})"
+      %ul
+        - @user.reviews.each do |review|
+          %article.review
+            %li.row
+              .col-sm-2
+                %p
+                  = link_to "\"#{review.video.title}\""
```

Comment:
You can remove the need for escaped quotes by using `%Q`:

```ruby
= link_to %Q("#{review.video.title}")
```

`%q` is like using single quotes, and `%Q` is like using double quotes, interpolation is allowed. Fun fact: `%` can also be used, without the `Q`. 

http://stackoverflow.com/questions/19782799/difference-between-q-q-in-ruby-string-delimiters